app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Api
+  module V1
+    class PostsController < ApplicationController
+      before_action :authenticate_user!
+      def index
+        @posts = Post.all
+
+        render json: {
+          data: @posts,
+          status: 200,
+          message: 'All post'
+        }, status: :ok
+      end
+
+      def create
+        @post = current_user.posts.new(post_params)
+
+        if @post.save
+          render json: {
+            status: :created,
+            message: 'Post was successfully created.'
+          }, status: :created
+
+        else
+          render json: {
+            status: :created,
+            message: 'Post was not created.'
+          }, status: :created
+        end
+      end
+
+      private
+
+      def post_params
+        params.require(:post).permit(:content)
+      end
+    end
+  end
```

Comment:
- [ ] I was unable to make requests for posts. I used two different API testing tools (postman and request maker). It always returns `401` error

| Request | Response |
|-------------|---------------|
| ![Screenshot from 2021-11-06 02-45-48](https://user-images.githubusercontent.com/1642014/140590183-554cc074-2e58-470e-bb1d-499998e0a514.png) | ![Screenshot from 2021-11-06 02-45-25](https://user-images.githubusercontent.com/1642014/140590195-a28accb8-90cf-4b6e-80ca-51b415ddb14c.png) |

Reply:
- Thank you @sinansevgi , 
But it looks like you are trying to make a GET request and not a post.
Please will you be more specific and help provide more clarity on what is not working with the application, because we tried it here is working
Thank you

Reply:
Hi @DanSam5K, I tried both `POST` and `GET` requests to all post-related endpoints.